sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/resourcemanager/AzureServiceResourceManagerConfigurationBase.java

```diff
@@ -18,7 +18,7 @@ public abstract class AzureServiceResourceManagerConfigurationBase {
      * Create {@link AzureServiceResourceManagerConfigurationBase} instance
      * @param azureResourceManager the azure resource manager
      */
-    public AzureServiceResourceManagerConfigurationBase(AzureResourceManager azureResourceManager) {
+    protected AzureServiceResourceManagerConfigurationBase(AzureResourceManager azureResourceManager) {
```

Comment:
this is part of API, can we just change it?

Reply:
will keep this public.